pkgs/development/tools/memray/default.nix

```diff
@@ -3,19 +3,20 @@
 , libunwind
 , lz4
 , pkg-config
-, python3
+, elfutils
+, python311Packages
 }:
 
-python3.pkgs.buildPythonApplication rec {
+python311Packages.buildPythonApplication rec {
```

Comment:
Is it necessary to go for `311`?

Reply:
`error: tree-sitter0_21-0.21.3 not supported for interpreter python3.12`